src/GPU/pppm_gpu.cpp

```diff
@@ -102,8 +102,6 @@ PPPMGPU::PPPMGPU(LAMMPS *lmp) : PPPM(lmp)
 PPPMGPU::~PPPMGPU()
 {
   PPPM_GPU_API(clear)(poisson_time);
-  destroy_3d_offset(density_brick_gpu,nzlo_out,nylo_out);
```

Comment:
This will introduce a memory leak.

Reply:
This was a mistake. Cannot recall why these lines are removed here. I reverted the changes to the current version of pppm_gpu.cpp in the develop branch,